packages/components/tests/integration/components/hds/copy/button/index-test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | hds/copy/button/index', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it should render the default component variation with a CSS class that matches the component name', async function (assert) {
+    await render(
+      hbs`<Hds::Copy::Button id="test-copy-button" @text="copy" @clipboardText="#clipboardText" />`
+    );
+    assert.dom('#test-copy-button').hasClass('hds-copy-button');
+  });
+
+  test('it should only render an icon and also render an aria-label if isIconOnly is set to true', async function (assert) {
+    await render(
```

Comment:
I did not do it that way because that's not the default use of the component. Typically for tests I use the defaults and change only the thing I'm testing.